CHANGELOG.md

```diff
@@ -3,7 +3,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keepa Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# [Unreleased](https://github.com/mypurecloud/webrtc-stats-gatherer/compare/v9.0.5...HEAD)
+# [Unreleased](https://github.com/mypurecloud/webrtc-stats-gatherer/compare/v9.0....HEAD)
+### Fixed
+* [PCM-1946](https://inindca.atlassian.net/browse/PCM-1946) - Fix issue with initial stats check generating an error
```

Comment:
I believe this should be pointing to PCM-2058.

Reply:
good catch.